extensions/wikibase/src/org/openrefine/wikibase/schema/entityvalues/SuggestedPropertyIdValue.java

```diff
@@ -26,20 +26,33 @@
 
 import org.wikidata.wdtk.datamodel.helpers.ToString;
 import org.wikidata.wdtk.datamodel.interfaces.PropertyIdValue;
+import org.wikidata.wdtk.datamodel.interfaces.Value;
 
 public class SuggestedPropertyIdValue extends SuggestedEntityIdValue implements PropertyIdValue {
 
+    private Value suggestedValue;
+
     public SuggestedPropertyIdValue(String id, String siteIRI, String label) {
         super(id, siteIRI, label);
     }
 
+    public SuggestedPropertyIdValue(String id, String siteIRI, String label, Value suggestedValue) {
```

Comment:
I don't think this is necessary: ItemIdValues can already be returned independently from PropertyIdValue, which is more flexible. There could be warnings where we are including an ItemIdValue without an associated PropertyIdValue, so I would prefer not to couple them here.

Reply:
Reverted changes to SuggestedIdValue